.eslintignore

```diff
@@ -0,0 +1,6 @@
+node_modules
+dist
```

Comment:
They could be removed. Added here
https://github.com/sxzz/eslint-config/blob/8aa39cdbf54d54a67aadd3ece990e9cc2896e16b/packages/basic/index.js#L16-L32